docs/source/theory.rst

```diff
@@ -113,6 +113,19 @@ According to the latter, the rate :math:`k(T)` of a thermally activated process
 
 where :math:`k_0` is the pre-exponential factor, :math:`E_k` is the process activation energy, and :math:`T` is the temperature. 
 
+Radioactive decay
+^^^^^^^^^^^^^^^^^
+
+Radioactive decay can be included in FESTIM. Neglecting other terms (diffusion, trapping...), the temporal evolution of the concentration of a species :math:`c` is governed by:
+
+.. math::
+    :label: eq_radioactive_decay
+
+    \frac{\partial c}{ \partial t} = -\lambda c
+
+where :math:`\lambda` is the decay constant.
```

Comment:
```suggestion
where :math:`\lambda\,[\mathrm{s}^{-1}]` is the decay constant.
```

Reply:
Yes I was wondering about adding units then realised that units were never specified anywhere in the Theory section.

Reply:
They are in #808

Reply:
Right so maybe we would need to go through the whole theory section and make sure units are everywhere then

Reply:
Nvm I forgot that you added units everywhere! 🚀